### PR TITLE
Mccalluc/prettier test output

### PR DESCRIPTION
**Pull request by mccalluc**
Test source code and output is vastly more clear with pytest instead of unittest.

### CODE REVIEW

File: test_workflows.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        os.rename(download_path, target)
+    metadata = json.load(open(path))
+    expected_suffix = metadata['schema_type'] + '.json'
+    if not path.endswith(expected_suffix):
+        raise Exception(f'Expected to end with "{expected_suffix}".')
+    described_by = metadata['describedBy']
+    schema_url = urlparse(described_by)
+    cache_path = Path('schema-cache') / schema_url.hostname / schema_url.path[1:]
+    # Remove leading "/" from URL path component.
+    if not os.path.isfile(cache_path):
```
Comment (mruffalo) [inline test_workflows.py]: Could be `if not cache_path.is_file():`

File: test_workflows.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+# "*-*" to exclude "prov.json"
+hca_paths = sorted(glob('workflows/*/outputs-hca/actual/*-*'))
+indexing_paths = sorted(glob('workflows/*/outputs-indexing/actual/*'))
+all_actual_json_paths = sorted(glob('workflows/*/outputs-*/actual/*'))
+prov_paths = sorted(glob('workflows/*/outputs-hca/actual/prov.json'))
+
+
+def test_all_globs_match():
+    assert len(hca_paths)
```
[the remark this answers is not in the document]
Comment (mccalluc) [reply test_workflows.py]: I was trying to make it more clear what the assertion was about, but sounds like it just raises more questions for you, so I'll change.

File: test_workflows.py (hunk 1 above)
Comment (mccalluc) [reply test_workflows.py]: 👍

File: test_workflows.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mruffalo) [reply test_workflows.py]: It's clear either way that you want each of the lists to have at least one element -- it just made me wonder why you're calling `len` when the result is the same either way. It jumped out at me the same way as `if some_boolean_expression == True`.

**Pull request merged**